### ISSUE TITLE
information about trained models

### ISSUE THREAD

**Issue by pariasm** [Coldog2333/pytoflow#1]
Thank you for porting this work to pytorch. Could you provide more information about the trained models given in the folder toflow_models?

In particular, for the video denoising model (denoise.pkl):
1. In the original work there are 3 types of noise considered (Gaussian 15, Gaussian 25 and mixed noise). Your denoise.pkl, on which type of noise was it trained?
2. Is it the one trained by the original authors or the one you retrained?
3. If it is the one you retrained, which trainset did you use, and for how many epochs.

**Comment by Coldog2333** [comment]
Thanks for being interested in pytoflow.

As for your proposed three questions:
1. I remember that it was trained on the dataset with mixed noise, but I am not sure because it has been done for almost half a year. If it is important to your project, you can check the pytoflow/generate_testing_sample/addnoise.m to figure out the answer. I used this matlab script to process the original Vimeo-90K at that time.
2. All of these models are retrained.
3. The experiment settings including the used trainset, the super-parameters (epochs, learning rate, etc.), the pipeline, are the same as the original paper. The only exception is that we use lr=3\*1e-5 in interpolation task while the learning rate in the original paper is 3*1e-4.

If you have other questions about this repository, feel free to contact me.
